composer.json

```
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=5.6"
+        "php": ">=5.6",
+        "laravel/framework": "^5.4|^6.0"
```

To make it easier to extend (to stuff like Laravel Lumen), I'd suggest using the [illuminate Support package](https://github.com/illuminate/support). This contains the service provider, but doesn't require the whole Laravel package (and the `laravel/framework` package will tell composer it replaces it.)

```suggestion
        "illuminate/support": "^5.4|^6.0"
```